packages/now-cli/test/dev/integration.js

```diff
@@ -153,6 +154,9 @@ async function testPath(
   if (typeof expectedText === 'string') {
     const actualText = await res.text();
     t.is(actualText.trim(), expectedText.trim(), msg);
+  } else if (typeof expectedText === 'function') {
```

Comment:
Added functionality to pass a validation function to `testPath()` for more complicated assertions.